regulations/static/regulations/css/less/module/comment.less

```diff
@@ -6,7 +6,30 @@ comment.less contains styles related to commenting on sections
 
 #preamble-read {
   margin: 0;
-  padding: 0;
+  padding: 0 0 0 15px;
+
+  li {
+    h2, h3, h4, h5, p {
+      width: 70%;
+    }
+  }
+
+  // neutralize padding for write comment link placing
+  .level-1 > li {
+    padding-top: 0;
+  }
+
+  // for paragraphs, only show write comment link when hovered
+  li:hover {
+    > p + .activate-write {
+      top: 15px;
+      display: block;
+    }
+  }
+
+  p + .activate-write {
+    display: none;
+  }
```

Comment:
The logic here needs to work a bit differently for CFR changes -- there one can comment on _any_ changing paragraph. We're also expecting to "call out" specific paragraphs of the preamble that have requests for comment.

#218 takes some strides in this direction, in particular, adding the variables "accepts_comments" and "comments_calledout". The latter should indicate when a paragraph should have the link to the side when _not_ hovering.